### DISCUSSION TITLE
This is an interface question.

### DISCUSSION THREAD

**Discussion by um38s** [cables-gl/cables_docs#301] [Answers & Questions]
Hello. I am learning programming for the first time. I can't find the red dotted line at the bottom of the screen on the right of the screen. Is there a shortcut that I can find?
![pic](https://user-images.githubusercontent.com/15072212/181041307-b32c10a5-6808-4b9d-9b57-a77be0090c32.png)

**Comment by steam0r** [comment]
click on the renderer (where you see the cube) and that menu will appear at the bottom of that section of the editor

**Reply by um38s** [reply]
Thank you.   Let's check.  :)

**Comment by pandrr** [comment]
it seems to be cut off by your small browserwindow.
if your screen is to small, try ctrl+p and type "reset canvas" / ctrl+p "canvas size" to set it manually
